README.md

```diff
@@ -241,7 +241,18 @@ We use [env_logger](https://crates.io/crates/env_logger): set the `RUST_LOG` env
 
 ### Unit tests
 
-`make test` - open the Makefile to adjust your `SYNC_SYNCSTORAGE__DATABASE_URL` as needed.
+You'll need [`nextest`](https://nexte.st/docs/installation/from-source/) and [`llvm-cov`](https://github.com/taiki-e/cargo-llvm-cov?tab=readme-ov-file#installation) installed for full unittest and test coverage.
+
+    $ cargo install cargo-nextest --locked
+    $ cargo +stable install cargo-llvm-cov --locked
```

Comment:
nit: I'm not sure why their install instructions explicitly mention `+stable` but that's the default toolchain for most users. it's less confusing without it IMHO
```suggestion
    $ cargo install cargo-llvm-cov --locked
```